htdocs/stripe/payout.php

```diff
@@ -154,15 +154,8 @@
 			{
 				$memberstatic->id = 0;
 			}*/
-
 			$societestatic->fetch($charge->metadata->idcustomer);
 			$societestatic->id = $charge->metadata->idcustomer;
```

Comment:
Just before, we have 
$societestatic->fetch()
It means object is loaded with all its property including the ->id set to $charge->metadata->idcustomer that is the id received as parameter to make the fetch.
So you can also remove this line   $societestatic->id = $charge->metadata->idcustomer;